omod/src/test/resources/registration_app.json

```diff
@@ -0,0 +1,176 @@
+[
+  {
+    "id": "referenceapplication.registrationapp.registerPatient",
+    "instanceOf": "registrationapp.registerPatient",
```

Comment:
@mks-d , is it fine duplicating this json resource(seems more clear)? Or just doing something like `RegistrationSummaryExtensionsGeneratorTest.getClassLoader().getResourceAsStream("registration_app.json");` when needed would work fine?

Reply:
I'm not sure if I understood correctly, but as long as this one test resource is good enough for any test that you're working on, use it ;-)